Artesian/Artesian.SDK/Dto/Auction/AuctionBids.cs

```diff
@@ -33,13 +33,13 @@ public AuctionBids(LocalDateTime bidTimestamp, AuctionBidValue[] bid, AuctionBid
         /// </summary>
         [Required]
         [Key(1)]
-        public AuctionBidValue[] Bid { get; set; }
+        public AuctionBidValue[]? Bid { get; set; }
```

Comment:
this is marked required. add ctor or default to empty array.

apply this to similar cases